spec/unit/classes/firewall_linux_archlinux_spec.rb

```diff
@@ -3,23 +3,17 @@
 require 'spec_helper'
 
 describe 'firewall::linux::archlinux', type: :class do
-  let(:facts) do
```

Comment:
all of the changes here aren't related to the resource_api I think? Should they be pulled out as well?

Reply:
Somewhat same as above, part of the update process was down as doing a full rubocop of the module to confirm the new code matched standards, and the autofixes brought in a large amount of fixes.

Reply:
mhm I understand that, but I would prefer it to have the rubocop changes in another PR. half of the diff is rubocop related.

Reply:
You have a point, will se what I can do.
Was thinking of doing a full rebase and squish anyway, my commits kinda got away from me.

Reply:
Ok, that's all the unrelated rubocop fixes taken out and all the commits squished and cleaned up.
Will put up a second PR once this one gets merged.